### ISSUE TITLE
viz limit on #atoms in jupyterlab

### ISSUE THREAD

**Issue by scottwsides** [nglviewer/nglview#987]
I've seen a similar issue as people were seeing with just the jupyter notebook with the number of atoms viewed with nglview. Increasing the IOdata rate seemed to be the solution that works for everybody. However, I'm using this fix for jupyterlab and I'm still seeing a small limit on the number of atoms viewed (~50,000 atoms). Should 'jupyter lab' be very different from 'jupyter notebook' ?

**Comment by scottwsides** [comment]
I've tried using aseio to convert positions and element symbols to an IOString that nglview turns into a view but around 60K atoms lots of atoms are getting clipped. I've tried ASE Atoms list and the same issue. I'm beginning to think that ASE is the issue instead of NGL. Whats the best underlying driver to use with NGLView when 100ks of atoms are being used.

**Comment by scottwsides** [comment]
I've realized some extra behavior of this error. I have methods that return an NGLview object (eg return nglview.show_ase(atoms)). Above a certain large number of atoms, a call to this method (inside a jupyterlab cell) is corrupted on the first call. When I execute the cell again... the viz seems to be ok. I've tried many, many workarounds, different versions of NGLview etc and nothing will fix this.  Please, any suggestions would be very helpful.

**Comment by hainm** [comment]
hi @scottwsides: We will investigate this issue. And sorry for the late reply, github does not send notification to my email anymore for some reasons.

**Comment by hainm** [comment]
> Above a certain large number of atoms, 

@scottwsides Do you have code example with public file to reproduce the issue? thanks.

**Comment by scottwsides** [comment]
Hi Hai, sorry to take so long. I finally found a workaround so I didn't get back right away. Sorry I no longer have the exact code that fails but I can describe it.  I was taking an xyz format string and then

`import io
from ase import io as aseio
fobj = io.StringIO(xyzStr)
mol = aseio.read(fobj, format='xyz')
view = nglview.show_ase(mol)
`
and then when visualizing the view... it would cut off atoms if the number got to around >100K atoms

What works is creating a list of Atoms( ....) objects, ase_atoms, then doing
`from ase.visualize import view
 v = view(ase_atoms, viewer='ngl')
` 

The first method is a bit convoluted... but does work for small atom numbers and it made sense considering the class I was using it in. I'm pretty sure it wasnt memory or nglview version or configuration. I tried LOTS of other variations... but the 2nd method above seemed to do the trick.
